certbot/client.py

```diff
@@ -313,6 +310,25 @@ def obtain_certificate(self, domains):
 
             return cert, chain, key, csr
 
+    def _get_order_and_authorizations(self, csr_pem, best_effort):
+        """Request an new order and complete its authorizations.
+
+        :param str csr_pem: A CSR in PEM format.
+        :param bool best_effort: True if failing to complete all
+            authorizations should raise an exception
```

Comment:
True if failing to complete all authorizations should **not** raise an exception?